tests/go/GoMetaLintBearTest.py

```diff
@@ -0,0 +1,115 @@
+import os
+
+from bears.go.GoMetaLintBear import GoMetaLintBear
+from queue import Queue
+from unittest.case import skipIf
+from shutil import which
+
+from coalib.testing.LocalBearTestHelper import execute_bear
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+
+
+def get_testfile_path(testfile):
+    return os.path.join(os.path.dirname(__file__),
+                        'gometalinter_test_files',
+                        testfile)
+
+
+def load_testfile(filename):
+    with open(get_testfile_path(filename)) as file:
+        contents = file.read().splitlines(True)
+    return contents
+
+
+@skipIf(which('go') is None, 'go is not installed')
+class GoMetaLintBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.section = Section('GoMetaLinter')
```

Comment:
Sorry for nitpicking. For consistency with the other test cases, I think it's better to not name the Section. So just `Section()` should be enough.